css/styles.css

```diff
@@ -108,3 +164,209 @@ body {
   display: flex;
   padding: 1px;
 }
+
+.work-section {
+  display: flex;
+  padding-top: 40px;
+  justify-content: center;
+}
+
+.title {
+  font-family: 'Crete Round', Arial, sans-serif;
+  font-style: normal;
+  font-weight: 200;
+  font-size: 40px;
+  line-height: 52px;
+  margin-left: 24px;
+  border-bottom: 1px solid rgb(0, 0, 0);
+}
+
+.my-works {
+  display: flex;
+  flex-direction: column;
+  padding: 3px;
+  padding-left: 15px;
+}
+
+.work-pic-1 img {
+  margin-top: 50px;
+  margin-bottom: 10px;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;

```

Comment:
As asked by the previous reviewer, you have to change the size of your image.
Here you give a value of 50% that's why it doesn't stick to Figma,
Please change 50% by 100% to have it like in the Template 

<img width="445" alt="image" src="https://user-images.githubusercontent.com/7243132/170840310-28f43d96-cc8a-4f80-8cf5-19ee6c5bd932.png">